src/clientfactory.ts

```diff
@@ -42,11 +42,18 @@ export class DiscordClientFactory {
         this.botClient = new DiscordClient({
             fetchAllMembers: true,
             messageCacheLifetime: 5,
-            sync: true,
```

Comment:
You didn't specify which itnents to use at all, so this won't work after nov 7th

Reply:
`new Discord.Client({ ws: { intents: Discord.Intents.NON_PRIVILEGED }})`

You might want to add a setting for if it'll use priveleged or unpriveleged intents